docs/src/main/sphinx/release/release-353.md

```diff
@@ -0,0 +1,87 @@
+# Release 353 (XXXXXXXX March 2021)
+
+## General
+
+* Fix aliasing of columns in common table expressions. ({issue}`6839`)
```

Comment:
per https://github.com/trinodb/trino/issues/6858#issuecomment-779728365 my not be end-user visible since no connector was actually affected

cc @sopel39 @kasiafi

Reply:
It could apply to third-party connectors

Reply:
This should also describe what this is fixing. Is it a failure? Incorrect results?

Reply:
Potentially, wrong columns were aliased and exposed from the CTE.
It could lead both to failure from type mismatch and incorrect query results (if the off columns matched the context of usage)